lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -114,12 +115,20 @@ class UnusedBytes extends Audit {
       settings,
     };
 
+    const gatherMode = artifacts.GatherContext && artifacts.GatherContext.gatherMode
+      || 'navigation';
     return NetworkRecords.request(devtoolsLog, context)
       .then(networkRecords =>
         Promise.all([
           this.audit_(artifacts, networkRecords, context),
-          PageDependencyGraph.request({trace, devtoolsLog}, context),
-          LoadSimulator.request(simulatorOptions, context),
+          PageDependencyGraph.request({trace, devtoolsLog}, context).catch(err => {
```

Comment:
as a style thing, it would be nice to put a lot of this stuff closer to top level where appropriate so you don't have to dig to see where mode differentiation is happening